docs/UPP_GRIB2_Table_byID.csv

```diff
@@ -0,0 +1,868 @@
+**Hydrophilic black carbon on model surface**,BCPHILIC_ON_HYBRID_LVL,PMTF,643
+**Air density on model surface**,AIR_DENSITY_ON_HYBRID_LVL,DEN,644
+**Layer thickness on model surface**,LAYER_THICKNESS_ON_HYBRID_LVL,THICK,645
+**Dust PM2.5 column mass density**,DUST_COL_MASS_DEN,COLMD,646
+**Sea salt PM2.5 column mass density**,SEAS_COL_MASS_DEN,COLMD,647
+**Aerosol single scattering albedo at 340 nm**,SINGLE_SCAT_ALBD_at340,SSALBK,648
+**Aersol asymmetry factor at 340nm**,AER_ASYM_FACTOR_at340,ASYSFK,649
+**Total aerosol scattering optical depth at 550nm**,AER_SCAT_OPT_DEP_at550,SCTAOTK,650
+**Dust aerosol scattering optical depth at 550nm**,DUST_AER_SCAT_OPT_DEP_at550,SCTAOTK,651
+**Seasalt aerosol scattering optical depth at 550nm**,SEASALT_AER_SCAT_OPT_DEP_at550,SCTAOTK,652
+**Sulfate aerosol scattering optical depth at 550nm**,SULFATE_AER_SCAT_OPT_DEP_at550,SCTAOTK,653
+**Organic carbon aerosol scattering optical depth at 550nm**,ORGANIC_CARBON_AER_SCAT_OPT_DEP_at550,SCTAOTK,654
+**Black carbon aerosol scattering optical depth at 550nm**,BLACK_CARBON_AER_SCAT_OPT_DEP_at550,SCTAOTK,655
+**Angstrom exponent**,ANGSTROM_EXP_at440_860,ANGSTEXP,656
+**NO3 column mass density**,NO3_COL_MASS_DEN,COLMD,657
+**NH4 column mass density**,NH4_COL_MASS_DEN,COLMD,658
+**Dust emission flux**,DUST_EMISSION_FLUX,AEMFLX,659
+**Dust sedimentation flux**,DUST_SEDIMENTATION_FLUX,SEDMFLX,660
+**Dust dry deposition flux**,DUST DRY DEPOSITION,DDMFLX,661
+**Dust wet deposition flux**,DUST WET DEPOSITION,WLSMFLX,662
+**Seasalt emission flux**,SEASALT_EMISSION_FLUX,AEMFLX,663
+**Seasalt sedimentation flux**,SEASALT_SEDIMENTATION_FLUX,SEDMFLX,664
+**Seasalt dry deposition flux**,SEASALT_DRY_DEPOSITION_FLUX,DDMFLX,665
+**Seasalt wet deposition flux**,SEASALT_WET_DEPOSITION_FLUX,WLSMFLX,666
+**Black carbon emission flux**,BLACK_CARBON_EMISSION_FLUX,AEMFLX,667
+**Black carbon sedimentation flux**,BLACK_CARBON_SEDIMENTATION_FLUX,SEDMFLX,668
+**Black carbon dry deposition flux**,BLACK_CARBON_DRY_DEPOSITION_FLUX,DDMFLX,669
+**Black carbon wet deposition flux**,BLACK_CARBON_WET_DEPOSITION_FLUX,WLSMFLX,670
+**Organic carbon emission flux**,ORGANIC_CARBON_EMISSION_FLUX,AEMFLX,671
+**Organic carbon sedimentation flux**,ORGANIC_CARBON_SEDIMENTATION_FLUX,SEDMFLX,672
+**Organic carbon dry deposition flux**,ORGANIC_CARBON_DRY_DEPOSITION_FLUX,DDMFLX,673
+**Organic carbon wet deposition flux**,ORGANIC_CARBON_WET_DEPOSITION_FLUX,WLSMFLX,674
+**Sulfate emission flux**,SULFATE_EMISSION_FLUX,AEMFLX,675
+**Sulfate sedimentation flux**,SULFATE_SEDIMENTATION_FLUX,SEDMFLX,676
+**Sulfate dry deposition flux**,SULFATE_DRY_DEPOSITION_FLUX,DDMFLX,677
+**Sulfate wet deposition flux**,SULFATE_WET_DEPOSITION_FLUX,WLSMFLX,678
+**Dust scavenging flux**,DUST_SCAVENGING_FLUX,WDCPMFLX,679
+**Seasalt scavenging flux**,SEASALT_SCAVENGING_FLUX,WDCPMFLX,680
+**Black carbon scavenging flux**,BLACK_CARBON_SCAVENGING_FLUX,WDCPMFLX,681
+**Organic carbon scavenging flux**,ORGANIC_CARBON_SCAVENGING_FLUX,WDCPMFLX,682
```

Comment:
for 632-682 could add "emission flux" > "emission mass flux"

Reply:
Should I only add "mass" for ones where it says "emission flux" or anywhere from 632-82 where I see "flux"?

Reply:
@gspetro-NOAA they all appeared to be mass fluxes, sorry for the confusion.

Reply:
@From what I can tell, I think you meant line 632 (corresponding to UPP ID 659) up through line 655 (corresponding to UPP ID 682)? I added "mass" to all of those. It didn't seem to make sense to do that for others before and after, but let me know if I missed anything.

Reply:
yes! oops